sql/storedprocedure.go

```diff
@@ -193,6 +184,47 @@ func (p *Procedure) ParamIndex(name string) int {
 	return -1
 }
 
+// Resolved implements the sql.Node interface.
+func (p *Procedure) Resolved() bool {
```

Comment:
This kind of functionality generally belongs in the plan package. Resist putting anything other than definitional types in the sql package, too much depends on it already